docker/compose.devcontainer.yaml

```
@@ -161,11 +161,32 @@ services:
+        target: /data
+        type: volume
 volumes:
   api_bash_history:
   api_node_modules:
   api_pnpm_store:
   cloudbeaver_data:
   minio_test_data:
   postgres_test_data:
+  redis_test_data:
+
+  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)